src/gui/mzroll/gallerywidget.cpp

```diff
@@ -329,56 +329,60 @@ void GalleryWidget::wheelEvent(QWheelEvent *event) {
 		}
```

Comment:
Did we make any programmatic change in this file at all? If not please try to not put code formatting changes in the same commit as those that are supposed to serve as fixes or enhancements (such as this one).

Reply:
Oh and the reason to not do this is, that it makes it difficult for me to review these commits. It gives me the impression that some logic was changed and I would go over it without realizing that its just a stylistic change.

Reply:
Okkay, I may have added it because of some printing statements i used to debug. But surely, will take care in future